content/docs/studio/rest-api.md

```diff
@@ -103,6 +103,143 @@ Running this example will fail because it requires that you have [set up remote
 cloud credentials] in DVC Studio to where the model is stored.
 
 </admon>
+</toggle>
+
+## Create new project
+
+To create a new project programmatically, you can integrate with the DVC Studio
+API. The API provides a POST request endpoint for creating projects.
+Authentication is required using a token, which can be generated from the user's
+profile settings in DVC Studio
+
+```yaml
+Endpoint: api/create-project
+HTTP Method: POST
+Content-Type: application/json
+```
+
+### Request
+
+Here are the possible values for each parameter in the request json:
+
+- `repo_url` (required): The URL of the repository where the project will be
```

Comment:
please review other docs in terms of style. we don't use `:` I think, we start lowercase, `required` - etc - probably there is also a convention around it